Firestore/Swift/Source/Codable/DocumentReference+ReadDecodable.swift

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2021 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+import FirebaseFirestore
+
+extension DocumentReference {
+  /// Encodes an instance of `Encodable` and overwrites the encoded data
```

Comment:
This comment doesn't match what the code does. Seems to have been copied from `setData`.

Reply:
You're correct - I am in the process of updating.